packages/react/src/components/DescriptionList/index.test.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import {
+  DescriptionList,
+  DescriptionListItem,
+  DescriptionTerm,
+  DescriptionDetails
+} from './index';
+import axe from '../../axe';
+
+describe('DescriptionList components', () => {
+  test('renders a dl element', () => {
+    const { container } = render(<DescriptionList>a</DescriptionList>);
+    const dl = container.querySelector('dl');
+    expect(dl).toBeInTheDocument();
```

Comment:
RTL typically recommends using a test id for instances like this:

```jsx
render(<DescriptionList data-testid="dl">a</DescriptionList>)
expect(screen.getByTestTestId('dl').tagName).toBe('DL')
```

Reply:
Thank you for the snippet! I wasn't aware of the tagname property until now. It's helpful for identifying HTML tags that don't have an assigned ARIA role.